From 718ee16f2cf77d94bc14989207b647cf6185b881 Mon Sep 17 00:00:00 2001
From: Song Liu <song@kernel.org>
Date: Fri, 20 Jan 2023 10:29:29 -0800
Subject: [PATCH] dummy patch

---
 kernel/bpf/syscall.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kernel/bpf/syscall.c b/kernel/bpf/syscall.c
index 11df90962101..bebf483390f1 100644
--- a/kernel/bpf/syscall.c
+++ b/kernel/bpf/syscall.c
@@ -1955,6 +1955,8 @@ static void bpf_audit_prog(const struct bpf_prog *prog, unsigned int op)
 	struct audit_context *ctx = NULL;
 	struct audit_buffer *ab;
 
+	if (!prog)
+		return;
 	if (WARN_ON_ONCE(op >= BPF_AUDIT_MAX))
 		return;
 	if (audit_enabled == AUDIT_OFF)
-- 
2.30.2

